knx/dpt/types.go

```diff
@@ -316,3 +316,211 @@ func (d DPT_12001) Unit() string {
 func (d DPT_12001) String() string {
 	return fmt.Sprintf("%d", uint32(d))
 }
+
+// DPT_13001 represents DPT 13.001 / counter value.
+type DPT_13001 int32
+
+func (d DPT_13001) Pack() []byte {
+	return packV32(int32(d))
+}
+
+func (d *DPT_13001) Unpack(data []byte) error {
+	var value int32
+
+	if err := unpackV32(data, &value); err != nil {
+		return err
+	}
+	*d = DPT_13001(value)
+
+	return nil
+}
+
+func (d DPT_13001) Unit() string {
+	return "pulses"
+}
+
+func (d DPT_13001) String() string {
+	return fmt.Sprintf("%d pulses", int32(d))
+}
+
+// DPT_13002 represents DPT 13.002 / flow rate.
+type DPT_13002 int32
+
+func (d DPT_13002) Pack() []byte {
+	return packV32(int32(d))
+}
+
+func (d *DPT_13002) Unpack(data []byte) error {
+	var value int32
+
+	if err := unpackV32(data, &value); err != nil {
+		return err
+	}
+	*d = DPT_13002(value)
+
+	return nil
+}
```

Comment:
This can be simplified analog to `func (d *DPT_13001) Unpack(data []byte) error`.